charts/agent-control-deployment/tests/configmap_agentcontrol_config_test.yaml

```diff
@@ -75,6 +75,8 @@ tests:
                 token_url: https://system-identity-oauth.service.newrelic.com/oauth2/token
               endpoint: https://opamp.service.newrelic.com/v1/opamp
               fleet_id: abcefg
+              signature_validation:
```

Comment:
can we add a single test with the override?

Reply:
We already have the test for US and EU. I removed the possibility to configure the url. Did we want it besides having the "default hardcoded" values?